README.md

```diff
@@ -1,26 +1,62 @@
-# Salesforce Apex
+# Salesforce Apex Library
 
 [![CircleCI](https://circleci.com/gh/forcedotcom/salesforcedx-apex.svg?style=svg&circle-token=5869ea795e44e1b737f2f2a86fd51cdc2ac08629)](https://circleci.com/gh/forcedotcom/salesforcedx-apex)
 ![npm (scoped)](https://img.shields.io/npm/v/@salesforce/plugin-apex)
```

Comment:
Is this also a stale reference to plugin-apex?

Reply:
Yep, removed